ru-RU/preferences/zen-preferences.ftl

```diff
@@ -0,0 +1,161 @@
+
+pane-zen-looks-title = Взгляни и почувствуй
+category-zen-looks =
+    .tooltiptext = { pane-zen-looks-title }
+
+zen-warning-language = Изменение языка по умолчанию может облегчить веб-сайтам отслеживание вас.
+
+zen-look-and-feel-colors-header = Выберите цвет темы
+zen-look-and-feel-colors-description = Сделайте { -brand-short-name} таким, как вам нравится
+zen-look-and-feel-theme-toolbar =
+    .label = Сделать панель инструментов тематической с акцентным цветом
```

Comment:
"тематической" doesn't really reflect what "themed" means in the source translation.
It might be reworded as something like "Использовать цветовой акцент для темы панели инструментов" (not ideal but can't think of a better way now)